pkg/keeper/keeper.go

```diff
@@ -318,13 +318,12 @@ func (c *DefaultController) Sync() error {
 	prs := make(map[string]PullRequest)
 	if c.spc.SupportsGraphQL() {
 		for _, query := range c.config().Keeper.Queries {
-			q := query.Query()
-			results, err := graphQLSearch(c.spc.Query, c.logger, q, time.Time{}, time.Now())
+			results, err := bucketedGraphQLSearch(c.spc.Query, query, c.logger)
 			if err != nil && len(results) == 0 {
-				return fmt.Errorf("query %q, err: %v", q, err)
+				return errors.Wrap(err, "failed to perform GraphQL queries for PRs, no results returned")
```

Comment:
errors.Wrap is deprecated (as linting would tell you). Keep with fmt.Errorf, but use %w as described here https://pkg.go.dev/fmt#Errorf

https://github.com/pkg/errors

Reply:
Ah cool. Sorry, thought there would be an import clash with the standard lib errors but it uses the fmt package.
Will make the change.